addons/mail/static/tests/systray/systray_activity_menu_tests.js

```diff
@@ -125,5 +142,48 @@ QUnit.test('activity menu widget: activity menu with 3 records', function (asser
 
     activityMenu.destroy();
 });
+
+QUnit.test('activity menu widget: activity view icon', function (assert) {
+    /* Check that the widget will correctly create the link */
+    assert.expect(7);
+    var self = this;
+    var activityMenu = new ActivityMenu();
+    testUtils.addMockEnvironment(activityMenu, {
+        services: this.services,
+        mockRPC: function (route, args) {
+            if (args.method === 'systray_get_activities') {
+                return $.when(self.data['mail.activity.menu']['records']);
```

Comment:
You don't need to access `records` property with `['records']`.
`obj['records']` is equivalent to `obj.records`.
You keep it for `['mail.activity.menu']`, because there are dots in the property name.